src/backend/cpu/kernel/regions.hpp

```diff
@@ -98,78 +99,77 @@ static void setUnion(LabelNode<T>* x, LabelNode<T>* y)
 template<typename T>
 void regions(Array<T> out, const Array<char> in, af_connectivity connectivity)
 {
-    const af::dim4 in_dims = in.dims();
-    const char *in_ptr  = in.get();
-    T    *out_ptr = out.get();
+    const af::dim4 inDims = in.dims();
+    const char *inPtr  = in.get();
+    T *outPtr = out.get();
 
     // Map labels
-    typedef typename std::map<T, LabelNode<T>* > label_map_t;
-    typedef typename label_map_t::iterator label_map_iterator_t;
+    typedef typename std::unique_ptr< LabelNode<T> > UnqLabelPtr;
+    typedef typename std::map<T, UnqLabelPtr > LabelMap;
```

Comment:
It's easier (for me) to read because it uses the assignment style to define the alias, not because it is shorter.